### PR TITLE
Add and Update documentation and samples

### PR DESCRIPTION
**Pull request by trainerbill**
Create README for samples folder showing exactly how to install and run samples
Add invoice paging sample
Add comment to invoice search

### CODE REVIEW

File: samples/invoice/search.py [hunk 1]
```diff
@@ -1,18 +1,22 @@
 from paypalrestsdk import Invoice
 import logging
+import pprint
```
Comment (bluk) [inline samples/invoice/search.py]: Not sure if this is used in this file or `search_pages.py`

File: samples/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+export PAYPAL_MODE=sandbox   # sandbox or live
+export PAYPAL_CLIENT_ID=EBWKjlELKMYqRNQ6sYvFo64FtaRLRR5BdHEESmha49TM
+export PAYPAL_CLIENT_SECRET=EO422dn3gQLgDbuwqTjzrFgFtaRLRR5BdHEESmha49TM
+```
+
+## Run Sample
+There may be additional comments in the sample file that you must complete before running.
+
+```sh
+python2.7 samples/payment/create_with_paypal.py
```
Comment (bluk) [inline samples/README.md]: Maybe just leave it as `python` for the general case.

File: samples/invoice/search_pages.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+from paypalrestsdk import Invoice
+import logging
+import pprint
```
Comment (jaypatel512) [inline samples/invoice/search_pages.py]: Unused import

File: samples/invoice/search_pages.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    invoices = Invoice.search(options)
+
+    if invoices.success():  # return True or False
+        for inv in invoices.invoices:
+            my_invoices.append(inv.id)
+    else:
+        print(invoices.error)
+
+   
+print json.dumps(my_invoices, sort_keys=False, indent=4)
```
Comment (JosephDev) [inline samples/invoice/search_pages.py]: minor thing, but I think it's better to use `print` as function like `print()`.
it helps someone who uses python3

File: samples/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+```sh
+git clone https://github.com/paypal/PayPal-Python-SDK.git
+cd PayPal-Python-SDK
+pip install -e .
+```
+
+## Configuration
+```sh
+export PAYPAL_MODE=sandbox   # sandbox or live
+export PAYPAL_CLIENT_ID=EBWKjlELKMYqRNQ6sYvFo64FtaRLRR5BdHEESmha49TM
```
Comment (jaypatel512) [inline samples/README.md]: A comment of this sort would be nice for both CLIENT_ID and SECRET
```
# You must change this to your sandbox clientId
```

**Comment by trainerbill** [comment]
@bluk too many samples to get through and I am being pulled away.  This is rebased and ready to merge.

**Pull request merged**